services/ui-src/src/components/export/ExportedModalOverlayReportSection.test.tsx

```diff
@@ -1,137 +1,93 @@
-            content: "previous page",
-            props: {
-              to: "/wp/state-and-territory-specific-initiatives/instructions",
-              style: {
-                textDecoration: "underline",
-              },
-            },
-          },
-          {
-            type: "html",
-            content: " for detailed instructions.",
-          },
-        ],
-      },
-      addEntityButtonText: "Add initiative",
-      editEntityHint: 'Select "Edit" to complete the details.',
-      editEntityButtonText: "Edit name/topic",
-      readOnlyEntityButtonText: "View name/topic",
-      addEditModalAddTitle: "Add initiative",
-      addEditModalEditTitle: "Edit initiative",
-      deleteModalTitle: "Are you sure you want to delete this initiative?",
-      deleteModalConfirmButtonText: "Yes, delete initiative",
-      deleteModalWarning:
-        "Are you sure you want to proceed? You will lose all information entered for this initiative in the Work Plan.",
-      enterEntityDetailsButtonText: "Edit",
-      readOnlyEntityDetailsButtonText: "View",
-      dashboardTitle: "Initiative total count:",
-      countEntitiesInTitle: true,
-      tableHeader: "Initiative name <br/> Work Plan topic",
-      addEditModalHint:
-        "Provide the name of one initiative. You will be then be asked to complete details for this initiative including a description, evaluation plan and funding sources.",
-    } as ModalOverlayReportPageVerbiage,
-    entitySteps: [
-      {
-        stepType: EntityDetailsStepTypes.DEFINE_INITIATIVE,
-        stepName: "mock step name",
-        hint: "mock step hint",
-        modalForm: {
-          fields: [
-            {
-              id: "mock field id",
-              validation: "number",
-            },
-          ],
-        },
-      },
-    ] as OverlayModalPageShape[],
-  },
-} as Props;
+const entityStep: (string | FormField)[] = [
+  "mockType",
+  "mock plan name",
+  "mock hint",
+  { id: "mock-id", props: { label: "mock field 1" }, type: "", validation: "" },
+];
+
+const entity: EntityShape = {
+  id: "entity-id",
+  type: "initiative",
+  "mock-id": "mock value",
+  mockType: [
+    {
+      id: "step-entity-id",
+    },
+  ],
+};
 
 const testComponent = (
   <RouterWrappedComponent>
-    <ExportedModalOverlayReportSection {...defaultMockProps} />
+    <ExportedOverlayModalReportSection
+      section={section}
+      entityStep={entityStep}
+      entity={entity}
+    />
   </RouterWrappedComponent>
 );
 
-describe("ExportedModalOverlayReportSection", () => {
+describe("Test ExportedOverlayModalReportSection Component", () => {
+  beforeEach(() => {
+    render(testComponent);
+  });
+  test("Test ExportRETTable render", () => {
+    screen.debug();
+  });
+
+  test("should render status icon", async () => {
+    render(testComponent);
+    (renderStatusIcon as jest.Mock).mockReturnValue(HTMLElement);
+    expect(renderStatusIcon(true)).toEqual(HTMLElement);
+  });
```

Comment:
`render(testComponent)` runs in the beforeEach so I don't think you need to repeat it. 

this is one of those tests where it seems like the mock is doing too much 😅 
could you instead render the real thing and verify that the image is present, or the alt text is present or something?